kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -2089,6 +2091,60 @@ def __init__(self):
+        if not self.run_fencer_timestamp:
+            self._fencer_recovery_pending = False
+            return
+        try:
+            url, host_uuid = self._get_report_url_and_host_uuid()
+            if not url or not host_uuid:
+                return
+            ps_uuids = list(self.run_fencer_timestamp.keys())
+            logger.info('recovering %d fencers after kvmagent restart' % len(ps_uuids))
+            self.report_self_fencer_state_changed_to_mn(ps_uuids)
+            self._fencer_recovery_pending = False
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)